source/administration/changelog.md

```diff
@@ -4,11 +4,77 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.9
+
+Release Date 2019-03-16
+
+Mattermost v5.9.0 contains low to medium level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Bug Fixes
+
+ - Fixed an issue where emoji reactions did not appear on posts right away.
+ - Fixed an issue where the emoji `Recently Used` cleared entirely after logging out and back in.
+ - Fixed an issue where emoji not included in our list of text-based emoji were not rendered as jumboemoji.
+ - Fixed an issue where email notification setting in the webapp was out of sync with the mobile apps.
+ - Fixed an issue where a broken image displayed on login page if custom branding was enabled but no image had been uploaded.
+ - Fixed an issue where at-channel, at-all, at-here followed by a period were not highlighted as mentions.
+ - Fixed an issue where the Mattermost icon was pixelated in bookmark rendering on Google Chrome.
+ - Fixed an issue where **System Console > Users** page had broken user interface on narrow screens.
+ - Fixed an issue where at-channel notification showed incorrect number of timezones.
+ - Fixed an issue where leading whitespace with emoji affected emoji size so that they didn't render as jumboemoji.
+ - Fixed an issue where the System Console graphs did not load smoothly.
+ - Fixed an issue with inconsistent formatting in page header on **System Console > Notifications > Mobile Push**.
+ - Fixed an issue where invite tokens with a 48-hour expiry expired after 24 hours.
+ - Fixed an issue where a blank screen appeared when opening a group message channel from "More" modal using Enter key.
+ - Fixed an issue where Zoom plugin caused link metadata code to print warnings in the System Console.
+ 
+### Compatibility
+
+#### Breaking Changes
+
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and multi-factor authentication is enabled, ensure your users have upgraded to mobile app version 1.17 or later. Otherwise, users who have MFA enabled may not be able to log in successfully. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
+ - The public IP of the Mattermost application server is considered a reserved IP for additional security hardening in the context of untrusted external requests such as Open Graph metadata, webhooks or slash commands. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
+ - Mobile app version 1.13+ is required for servers 5.4+
```

Comment:
When do we remove the note about mobile app version 1.13+? It's not really a breaking change for 5.9, more like a compatbility note.

Should we have a minimum version requirement note for mobile app version 1.17+ when `DisableLegacyMfa` is enabled?

Reply:
Added to mobile changelog